integration_test/workflow_tests/test_packages/ingest/test_multiple_granules_happy_path.py

```diff
@@ -0,0 +1,220 @@
+import json
+import logging
+import os
+import time
+import uuid
+from unittest import TestCase, mock
+
+import boto3
+
+import helpers
+
+
+class TestMultipleGranulesHappyPath(TestCase):
```

Comment:
From https://nasa.github.io/cumulus-orca/docs/developer/development-guide/code/integration-tests#ingest

- Include a large file to test timeout limits
- Test ingesting from both Glacier and regular buckets

Reply:
https://bugs.earthdata.nasa.gov/browse/ORCA-629 for timeout task
https://bugs.earthdata.nasa.gov/browse/ORCA-630 for testing ingest from glacier bucket